app/models/candidate.rb

```diff
@@ -24,12 +24,16 @@ def set_defaults
     self.vote ||= false
   end
 
-  def show_specialties
-    return self.specialties.map{|s| s.name}.to_sentence
-  end
-
   def order
     peerage_type.to_sym
   end
 
+  def list
```

Comment:
How is this change related to specialties?

Reply:
It's in the specialties and groups candidate table.